CHANGELOG.md

```diff
@@ -16,13 +16,16 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 - Changed from deprecated db update method
 - Pre-selected fields to run queries with in dashboard page
 - Do not filter by any institute when first accessing the dashboard
+- Reworked the layout and content of cancer variant view
```

Comment:
It just came to my mind that it would be very informative for the users if you could here a list of all the panels that are gone from the cancer variant page!